scripts/in_container/bin/run_tmux

```diff
@@ -55,16 +55,16 @@ tmux send-keys -t 'Main' 'bash' C-m 'clear' C-m
 
 tmux split-window -v
 tmux select-pane -t 1
-tmux send-keys 'airflow scheduler' C-m
+tmux send-keys 'opentelemetry-instrument airflow scheduler' C-m
```

Comment:
This is again tricky (a bit). This "opentelemetry-instriument" should only be added when INTEGRATIONS environment variable will contain "open-telemetry". 

I think this should be something like:

```
            COMMAND_PREFIX=""
            if [[ " ${INTEGRATIONS[*]} " =~ " open-telemetry " ]]; then
               COMMAND_PREFIX="opentelemetry-instrument"
            fi
````

And then:

```
tmux send-keys "${COMMAND_PREFIX} airflow scheduler" C-m
```
(The " instead of ' are important)

But you will have to test it :)

Reply:
Our discussion:

```
COMMAND_PREFIX="" 
COMMAND_SUFFIX=""
if [[ " ${INTEGRATIONS[*]} " =~ " jaegger " ]]; then 
    COMMAND_PREFIX="opentelemetry-instrument" 
    COMMAND_SUFFIX="--opentelemetry"
fi
```
```
tmux send-keys "${COMMAND_PREFIX} airflow scheduler ${COMMAND_SUFFIX}" C-m
```

Reply:
And then (in webserver_command):

```
if args.opentelemetry:
     run_args = ['opentelemetry-instrument'] + run_args
```